src/utils/common.js

```
@@ -0,0 +1,14 @@
+// common.js
+
+export const formatPrice = (price) => {
+  // Ensure the input is a number
+  if (typeof price !== "number") {
+    price = parseFloat(price);
+    if (isNaN(price)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)